src/main/java/com/teragrep/jos_01/procfs/status/process/Stat.java

```diff
@@ -0,0 +1,396 @@
+        fields = new CharacterDelimited(origin, " ").read();
+        pid = Long.parseLong(fields.get(0));
+        comm = fields.get(1);
+        state = fields.get(2);
+        ppid = Long.parseLong(fields.get(3));
+        pgrp = Long.parseLong(fields.get(4));
+        session = Long.parseLong(fields.get(5));
+        tty_nr = Long.parseLong(fields.get(6));
+        tpgid = Long.parseLong(fields.get(7));
+        flags = Long.parseLong(fields.get(8));
+        minflt = Long.parseLong(fields.get(9));
+        cminflt = Long.parseLong(fields.get(10));
+        majflt = Long.parseLong(fields.get(11));
+        cmajflt = Long.parseLong(fields.get(12));
+        utime = Long.parseLong(fields.get(13));
+        stime = Long.parseLong(fields.get(14));
+        cutime = Long.parseLong(fields.get(15));
+        cstime = Long.parseLong(fields.get(16));
+        priority = Long.parseLong(fields.get(17));
+        nice = Long.parseLong(fields.get(18));
+        num_threads = Long.parseLong(fields.get(19));
+        itrealvalue = Long.parseLong(fields.get(20));
+        starttime = Long.parseLong(fields.get(21));
+        vsize = Long.parseLong(fields.get(22));
+        rss = Long.parseLong(fields.get(23));
+        rsslim = new BigInteger(fields.get(24));
+        startcode = Long.parseLong(fields.get(25));
+        endcode = Long.parseLong(fields.get(26));
+        startstack = Long.parseLong(fields.get(27));
+        kstkesp = Long.parseLong(fields.get(28));
+        kstkeip = Long.parseLong(fields.get(29));
+        signal = Long.parseLong(fields.get(30));
+        blocked = Long.parseLong(fields.get(31));
+        sigignore = Long.parseLong(fields.get(32));
+        sigcatch = Long.parseLong(fields.get(33));
+        wchan = Long.parseLong(fields.get(34));
+        nswap = Long.parseLong(fields.get(35));
+        cnswap = Long.parseLong(fields.get(36));
+        exit_signal = Long.parseLong(fields.get(37));
+        processor = Long.parseLong(fields.get(38));
+        rt_priority = Long.parseLong(fields.get(39));
+        policy = Long.parseLong(fields.get(40));
+        delayacct_blkio_ticks = Long.parseLong(fields.get(41));
+        guest_time = Long.parseLong(fields.get(42));
+        cguest_time = Long.parseLong(fields.get(43));
+        start_data = Long.parseLong(fields.get(44));
+        end_data = Long.parseLong(fields.get(45));
+        start_brk = Long.parseLong(fields.get(46));
+        arg_start = Long.parseLong(fields.get(47));
+        arg_end = Long.parseLong(fields.get(48));
+        env_start = Long.parseLong(fields.get(49));
+        env_end = Long.parseLong(fields.get(50));
+        exit_code = Long.parseLong(fields.get(51));
+        timestamp = origin.timestamp();
+    }
+
+    @Override
+    public ArrayList<String> read() {
+        return fields;
+    }
+
+    public Instant timestamp() {
+        return timestamp;
+    }
+
+    public void printTimestamp() {
+        LOGGER.info(Date.from(timestamp).toString());
```

Comment:
Date.from formats as 
```
dow mon dd hh:mm:ss zzz yyyy
```

which is in conflict with Procedure: Code Programming which states that ISO 8601 is a good format. Please change these

Reply:
I.e. do not use java.util.Date, use instead Instant.toString() which provides ISO 8601.